_inc/client/components/welcome-new-plan/monitor-akismet-backups-prompt.jsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate as __ } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { imagePath } from 'constants/urls';
+import InlineModuleToggle from 'components/module-settings/inline-module-toggle';
+
+class MonitorAkismetBackupsPrompt extends React.Component {
+	render() {
+		return <div>
+			<img src={ imagePath + 'security.svg' } className="jp-welcome__svg" alt={ __( 'Security' ) } />
+			<p>
+				{ __( 'Jetpack Professional gives you everything you need to keep your hard work safe, including ' +
```

Comment:
This is shown for all three plans. Perhaps we could say, "Your Jetpack plan...". Or, we can add a placeholder in and then substitute the plan name in.